src/pages/CityPage/CityPageGuides.jsx

```diff
@@ -0,0 +1,141 @@
+import { useState, memo } from 'react';
+import { Link } from 'react-router-dom';
+
+import GuideCard from '../../components/common/GuideCard/GuideCard';
+import CategoryCard from '../../components/common/CategoriesCards/CategoryCard';
+import CreateGuideButton from '../../components/common/CreateGuideButton/CreateGuideButton';
+
+import HistoricalIcon from '../../assets/img/CategoriesCards/Icons/HistoricalIcon.svg';
+import PhotoIcon from '../../assets/img/CategoriesCards/Icons/PhotoIcon.svg';
+import FoodIcon from '../../assets/img/CategoriesCards/Icons/FoodIcon.svg';
+import CultureIcon from '../../assets/img/CategoriesCards/Icons/CultureIcon.svg';
+import ParksIcon from '../../assets/img/CategoriesCards/Icons/ParksIcon.svg';
+import MarketIcon from '../../assets/img/CategoriesCards/Icons/MarketIcon.svg';
+
+import HistoricalLocations from '../../assets/img/CategoriesCards/HistoricalLocations.png';
+import PhotoLocations from '../../assets/img/CategoriesCards/PhotoLocations.png';
+import FoodTour from '../../assets/img/CategoriesCards/FoodTour.png';
+import CulturalEvents from '../../assets/img/CategoriesCards/CulturalEvents.png';
+import Parks from '../../assets/img/CategoriesCards/Parks.png';
+import Markets from '../../assets/img/CategoriesCards/Markets.png';
+import NoCategory from '../../assets/img/CategoriesCards/NoCategory.png';
+import Stub from '../../assets/img/cities/Stub.png';
+
+const CategoriesButtonsData = [
```

Comment:
The world would be better if you stored this in a separate file

Reply:
Initially it was like that, but at that moment we didn't discuss how the icons for categories would appear. And I realized that if I need to render a guide card by the condition if(categoryName==History) {then show the history icon} etc., then I will have to re-import the icons again, which will cause an increase in code. Now that I have an understanding that the icons will come from the backend (possibly), then I can  create a separate export for icons in the file with category information. export const Icons = {
  Historical: <HistoricalIcon />,
  Photo: <PhotoIcon />,
  Food: <FoodIcon />,
  Culture: <CultureIcon />,
  Parks: <ParksIcon />,
  Market: <MarketIcon />,
};